content/docs/start/data-and-model-versioning.md

```diff
@@ -13,7 +13,7 @@ and seeing data files and machine learning models in the workspace. Or switching
 to a different version of a 100Gb file in less than a second with a
 `git checkout`.
 
-The foundation of DVC consists of a few commands that you can run along with
+The foundation of DVC consists of a few commands which you can run along with
```

Comment:
What about just
```suggestion
The foundation of DVC consists of a few commands which can run along with
```
I'm just getting a feeling of "which" overuse. We can ask others to proof read if needed.

Reply:
Gonna commit this one to try and move along, feel free to go back to this.

Reply:
> Committed, please `git pull`.